study-group/Toast-Implementation-Studies.md

```diff
@@ -407,9 +407,108 @@ The library allows showing multiple toasts, and gives the developer specific con
 - Comes with default icons, can change by modifying CSS.
 - **Note**: On mobile there's currently no simple way to stop the timeout, accomplished by hover on desktop.
 
-## Bootstrap
+## [Bootstrap Toast](https://getbootstrap.com/docs/4.2/components/toasts/)
+The Bootstrap Toast is a component from the Bootstrap library to provide a lightweight, customizable notification in the library's distinct style.
 
-*TODO: Fill in study for this library*
+### Design & Code Samples
+The Toast component, like many others in the Bootstrap library, is put directly into HTML:
+
+```html
+<div class="toast" role="alert" aria-live="assertive" aria-atomic="true">
+  <div class="toast-header">
+    <img src="..." class="rounded mr-2" alt="...">
+    <strong class="mr-auto">A Bootstrap Toast</strong>
+    <button type="button" class="ml-2 mb-1 close" data-dismiss="toast" aria-label="Close">
+      <span aria-hidden="true">&times;</span>
+    </button>
+  </div>
+  <div class="toast-body">
+    Hello, world! This is a Bootstrap Toast!
+  </div>
+</div>
+```
+
+Bootstrap uses classes to denote special elements of the components it constructs, like the classes `toast`, `toast-header`, and `toast-body`.
+The example uses many `aria` attributes, indicating a deliberate effort for accessibility.
+Multiple Toasts can be wrarpped together in a `div` to stack intuitively.
```

Comment:
I'm not sure what "stack intuitively" means. Might be good to include images and now that I think of it, images in some of the other studies could be helpful too (I don't have specific examples but maybe having a snapshot of any interesting examples would be good. Nobody is likely to actually make one of these examples run to see what it looks like.

Reply:
That's a good idea, something I had discussed with Domenic was creating a file of informative snapshots to go along with the study and link relevant ones to sections in this doc. Do you think I should do that, or just add in a few to this doc?

Reply:
@fergald I'm going to make a branch once the content is all pushed in to clean it up, by editing and adding some relevant images.

Reply:
Yeah, it's not urgent but I think surveys of UI components will benefit from images (but this being mostly an API survey, not so much, mostly for demonstrating results of non-obvious APIs).